plugins/clamd.js

```diff
@@ -25,7 +25,7 @@ exports.load_excludes = function () {
                         new_skip_list_exclude.push(re);
                     }
                     catch (e) {
-                        plugin.logerror(e.message + ' (entry: ' + list[i] + ')');
+                        plugin.logerror(e.message + ` (entry: ${list[i]})`);
```

Comment:
any reason to leave `e.message` out of the template literal?

Reply:
Will put it back inside.